atlas/AtlasMl/atlasml/models/competency.py

```
@@ -41,30 +32,22 @@ class GenerateCompetencyRequestBatch(BaseModel):
 
 class SuggestCompetencyResponse(BaseModel):
     competencies: list[Competency]
-    competency_relations: list[CompetencyRelation]
-
 
 class SaveCompetencyRequest(BaseModel):
-    id: str
-    description: str
-    competencies: list[Competency]
-    competency_relations: list[CompetencyRelation]
+    competency: Optional[Competency] = None
```

Ruff (0.12.2)

Use `X | None` for type annotations

Convert to `X | None`

(UP045)